docker-compose.override-sites.initial.yml

```diff
@@ -0,0 +1,6 @@
+version: '2'
+
+services:
+  web:
+    extra_hosts:
+     - "default.web.mw.localhost:${IDELOCALHOST}"
```

Comment:
This is only doing the little docker network <-> host roundtrip to avoid the nginx port mismatch between the two. That will no longer be necessary once we can specify the nginx port, => https://github.com/nginx-proxy/nginx-proxy/commit/b4709639b3c73b6c074637e67bb615813dab6cc2